packages/editor/src/utils/index.js

```diff
@@ -2,5 +2,10 @@
  * Internal dependencies
  */
 import mediaUpload from './media-upload';
-
 export { mediaUpload };
+
+import hearthbeatPostLocking from './heartbeat-post-locking';
```

Comment:
Minor: we can write `export { default as hearthbeatPostLocking } from './hearbeat-post-locking'`;

Reply:
sure, makes sense